server/sonar-web/src/main/js/components/issue/views/assign-form-view.js

```diff
@@ -152,7 +152,7 @@ export default ActionOptionsView.extend({
   },
 
   makeUnique (assignees) {
-    return _.uniq(assignees, false, function (assignee) {
+    return _.uniq(assignees, false, assignee => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)